controllers/shooter.py

```diff
@@ -57,6 +58,7 @@ class ShooterController(StateMachine):
     AUTO_MAX_ACCEL = 0.2  # G
     AUTO_ALLOWABLE_TURRET_ERROR = 0.3  # m, ring is 1.22m diameter
 
+    _wants_to_clear = False
     _wants_to_fire = will_reset_to(False)
```

Comment:
What happens if both of these are true?

I think that is probably undesirable, and we should make that state impossible. For example we could have an enum of possible commands for the shooter.